packages/react-notion-x/src/components/header.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
```

Comment:
LGTM! But the PR will look better and less confused if change it in the `page-header` file while not creating a new one and deleting it 😁